### PR TITLE
magento-engcom/community-portal#240 Redesign Project Details Page

### PR DESCRIPTION
**Pull request by XiyaMagento**
magento-engcom/community-portal#240 Redesign Project Details Page

- [x] Align project details page design with profile page design
- [x] Remove table of project information
- [x] Add list of pledgers

### CODE REVIEW

**Comment by XiyaMagento** [comment]
Since we are showing the list of pledgers on the project card, do we still need to display them on the project details page?

File: frontend/src/components/ProjectDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    return (
+      <div>
+        <Typography className={classes.titleText}>
+          {this.props.project.name}
+        </Typography>
+        <Card className={classes.card}>
+          <CardContent className={classes.content}>
+            <Typography className={classes.descriptionText}>{this.props.project.description}</Typography>
+          </CardContent>
+        </Card>
```
Comment (vrann) [inline frontend/src/components/ProjectDetails.tsx]: would that display pledgers as well?

Comment (XiyaMagento) [reply frontend/src/components/ProjectDetails.tsx]: Actually I was a bit unsure about this part. Should we display the pledgers in the same way as we display them on the project card?

Comment (vrann) [reply frontend/src/components/ProjectDetails.tsx]: on the project details page it should show all pledgers, not just first 5 as on the card

Comment (XiyaMagento) [reply frontend/src/components/ProjectDetails.tsx]: Okay, I will work on adding that to this PR

Comment (XiyaMagento) [reply frontend/src/components/ProjectDetails.tsx]: I have added the list of pledgers to the project details page

**Pull request merged**